infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -144,6 +155,46 @@ public LogReplicationSinkManager(String localCorfuEndpoint, LogReplicationConfig
         init(metadataManager, config);
     }
 
+    /**
+     * In Log Replication table entries are sent to Standby in form of OpaqueEntry. Registry table's
```

Comment:
For the use cases, those two cases are basically the same since (at least for now) user could only open with a different is_federated flag during rolling upgrade. Also we are not only checking options or is_federated flag, instead we will filter out the entries as long as there is already a record for this table

Reply:
not true, right?  A user can reopen an already opened table with a different is_federated option on the same version.